consensus/state_test.go

```diff
@@ -618,238 +618,6 @@ func TestStateLockPOLRelockThenChangeLock(t *testing.T) {
 	ensureNewRound(newRoundCh, height+1, 0)
 }
 
-// 4 vals, one precommits, other 3 polka at next round, so we unlock and precomit the polka
```

Comment:
why these were removed?

Reply:
you're right - these shouldn't have been moved, just the specific lines relating to the evidence pool